### PR TITLE
Feature/new landing

### PR DESCRIPTION
**Pull request by DylanMoriarty**
Implements a new landing page for OpenAerialMap.

@danielfdsilva for an initial review

### CODE REVIEW

File: app/assets/scripts/main.js [hunk 1]
```diff
@@ -1,8 +1,80 @@
 'use strict';
 import config from './config';
 import OAM from 'oam-design-system';
+import $ from 'jquery'; window.$ = $;
```
Comment (danielfdsilva) [inline app/assets/scripts/main.js]: Can we avoid having jquery global?

File: app/assets/scripts/main.js [hunk 2]
```diff
@@ -1,8 +1,80 @@
...
+      targetEl.attr('href', constructUrl(imgData));
+      // Fade out loading spinner
+      $('.loading', targetEl).removeClass('revealed');
+    });
+  });
+};
+
+// Add Flexslider to Projects Section
+const createSlider = () => {
+  $('.flex-slider').flexslider({
```
Comment (danielfdsilva) [inline app/assets/scripts/main.js]: Flexslider should come from a node module.
https://www.npmjs.com/package/flexslider

File: app/assets/scripts/main.js [hunk 3]
```diff
@@ -1,8 +1,80 @@
...
+  const quadKey = tilebelt.tileToQuadkey(tile);
+  const mapView = center[0] + ',' + center[1] + ',' + previewZoom;
+  // Return OAM Browser URL including map view, tile, and image id
+  return `http://beta.openaerialmap.org/#/${mapView}/${quadKey}/${imgData._id}`;
+};
+
+// Update Latest Imagery containers to describe and link to most recent imagery
+const updateLatest = () => {
+  // Fetch metadata for the three most recent images, in descending order
+  const catalogueUrl = 'https://api.openaerialmap.org/meta?order_by=acquisition_end&sort=desc&limit=3';
```
Comment (danielfdsilva) [inline app/assets/scripts/main.js:55]: The base paths for these urls should be stored in the config file.

File: app/index.html [hunk 4]
```diff
@@ -50,55 +47,214 @@
...
+            <p>With each contribution our map of the world gets more complete.</p>
+            <ul class = "latest-imagery__cards">
+              <li id = "image-1" class = "latest-imagery__card">
+                <a href="http://beta.openaerialmap.org/">
+                  <section class="panel">
+                    <div class="latest-imagery__image">
+                      <div class="loading revealed"></div>
+                    </div>
+                    <footer class="panel__footer latest-imagery__details">
+                      <p>Uploaded: <strong class='uploaded' /></strong></p>
```
Comment (danielfdsilva) [inline app/index.html]: Self closing strong?

**Comment by smit1678** [comment]
To add a final to do list: 
- [x] Add text for Tools section
- [x] Add image and text for Contribute section
- [x] hook up View All button
- [x] @nbumbarger to hook up stats once we `/analytics` endpoints are up
- [x] review text

@danielfdsilva @DylanMoriarty @nbumbarger anything else here?

File: app/assets/styles/_landing-page-styles.scss [hunk 5]
```diff
@@ -293,10 +291,9 @@
...
 .community__text{
-	.inner{
-		p{
-		 @include center($max_width: 720px);
-		}
+	margin-bottom:6em;
+	p{
+	 @include center($max_width: 720px);
 	}
 }
```
Comment (danielfdsilva) [inline app/assets/styles/_landing-page-styles.scss]: Please mind the text indentation.
- Should be set to 2 spaces
- There should be a space between the text and `{` and between `:` and property value.

It helps with readability.

**Comment by nbumbarger** [comment]
@smit1678 The stats in the header connect to https://oam-catalog-test.herokuapp.com/analytics/?limit=1

The PR that adds these additional stats has been added to the official catalog API, but it isn't live yet. When it's ready, you'll want to replace the URL on [line 10 of the main.js script](https://github.com/hotosm/openaerialmap.org/blob/feature/new-landing/app/assets/scripts/main.js#L10 script) with `http://api.openaerialmap.org/analytics?limit=1`, before merging this PR.

**Pull request merged**